source/geometries/NextDemoSiPMBoard.cc

```diff
@@ -48,23 +51,26 @@ NextDemoSiPMBoard::NextDemoSiPMBoard():
   mask_thickn_     (2.0  * mm),
   membrane_thickn_ (0.),
   coating_thickn_  (0.),
+  hole_type_       (""),
   hole_diam_       (3.5  * mm),
+  hole_x_          (0.0  * mm),
+  hole_y_          (0.0  * mm),
+  sipm_type_       (""),
   mother_phys_     (nullptr),
   kapton_gen_      (nullptr)
 {
   msg_ = new G4GenericMessenger(this, "/Geometry/NextDemo/",
                                 "Control commands of the NextDemo geometry.");
 
-  msg_->DeclareProperty("sipm_board_verbosity", verbosity_,
-                        "NextDemoSiPMBoard verbosity");
+  msg_->DeclareProperty("sipm_board_verbosity",       verbosity_, "NextDemoSiPMBoard verbosity");
+  msg_->DeclareProperty("sipm_verbosity"      ,  sipm_verbosity_, "NextDemo SiPMs verbosity");
+  msg_->DeclareProperty("sipm_board_vis"      ,      visibility_, "NextDemoSiPMBoard visibility.");
+  msg_->DeclareProperty("sipm_visibility"     , sipm_visibility_, "SiPMs Visibility");
```

Comment:
For uniformity, I would use the same type of string as the other three parameters, namely: "NextDemo SiPMs visibility".